### PR TITLE
[Quantization] remove all q_val & q_info

### PR DESCRIPTION
**Pull request by chenbohua3**
Also apply linter to the codes

### CODE REVIEW

File: pytorch_blade/src/compiler/tensorrt/bridge/tensorrt_onnx_parser.cpp [hunk 1]
```diff
@@ -183,74 +176,5 @@ bool TensorrtOnnxParser::IsSupportedModel(const std::string& proto_bytes) {
   return GetSegmentList(proto_bytes, nodes_segment_list);
 }
 
-bool TensorrtOnnxParser::setDynamicRange(
```
Comment (fortianyou) [inline pytorch_blade/src/compiler/tensorrt/bridge/tensorrt_onnx_parser.cpp:186]: What is the difference between `setDynamicRange` directly and given calibration data? Could you please claim the decision choice you made?

File: pytorch_blade/src/compiler/tensorrt/bridge/tensorrt_onnx_parser.h [hunk 2]
```diff
@@ -43,15 +41,12 @@ class TensorrtOnnxParser {
...
-      const QValType&);
+      const std::vector<DynamicRanges>);
 
   void SetMaxWorkspaceSize(const size_t workspace_size) {
     max_workspace_size_ = workspace_size;
   }
-  bool setDynamicRange(
-      TrtUniquePtr<nvinfer1::INetworkDefinition>& network,
-      const QValType& q_val);
+  bool setDynamicRange(TrtUniquePtr<nvinfer1::INetworkDefinition>& network);
```
[the remark this answers is not in the document]
Comment (chenbohua3) [reply pytorch_blade/src/compiler/tensorrt/bridge/tensorrt_onnx_parser.h]: done

File: pytorch_blade/src/compiler/tensorrt/bridge/tensorrt_onnx_parser.cpp (hunk 1 above)
Comment (chenbohua3) [reply pytorch_blade/src/compiler/tensorrt/bridge/tensorrt_onnx_parser.cpp:186]: Have added some notes about the two ways of trt quantization

**Review by fortianyou — APPROVED** [review]
LGTM

**Pull request merged**